tools/autograd/gen_variable_type.py

```diff
@@ -89,6 +89,7 @@
     'reflection_pad1d_backward', 'reflection_pad2d_backward',
     'replication_pad1d', 'replication_pad2d', 'replication_pad3d',
     'replication_pad1d_backward', 'replication_pad2d_backward', 'replication_pad3d_backward',
+    'masked_scatter', 'masked_select'
```

Comment:
we shouldn't enable `masked_scatter` since it's dependent on `_th_masked_fill_bool_` which is not supported for complex